### PR TITLE
Add note clarifying the need for a switchbot hub.

### PR DESCRIPTION
**Pull request by tels7ar**
It's not super clear that you need a hub of some sort to integrate switchbot with home assistant.

## Proposed change

Clarify that you need a hub to use the SwitchBot integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/switchbot.markdown [hunk 1]
```diff
@@ -14,6 +14,15 @@ ha_platforms:
 
 The `switchbot` switch platform allows you to control SwitchBot [devices](https://www.switch-bot.com/).
 
+
+<div class="note">
+
+SwitchBot devices operate over Bluetooth Low Energy (BLE) and require some sort of hub to connect to Home Assistant.  That can either be the [official SwitchBot hub](https://www.switch-bot.com/products/switchbot-hub-mini) or something like [SwitchBot-MQTT-BLE-ESP32](https://github.com/devWaves/SwitchBot-MQTT-BLE-ESP32) running on an ESP32 board.
```
Comment (frenck) [inline source/_integrations/switchbot.markdown:20]:
```suggestion
SwitchBot devices operate over Bluetooth Low Energy (BLE) and require some sort of hub to connect to Home Assistant. That can either be the [official SwitchBot hub](https://www.switch-bot.com/products/switchbot-hub-mini) or something like [SwitchBot-MQTT-BLE-ESP32](https://github.com/devWaves/SwitchBot-MQTT-BLE-ESP32) running on an ESP32 board.
```

File: source/_integrations/switchbot.markdown [hunk 2]
```diff
@@ -14,6 +14,15 @@ ha_platforms:
...
+
+tl;dr: this integration won't work if you just buy a SwitchBot and nothing else.
+
```
Comment (frenck) [inline source/_integrations/switchbot.markdown]: A TL;DR on a 2 line note block, seems a bit overkill.

```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/switchbot.markdown (hunk 1 above)
Comment (Danielhiversen) [reply source/_integrations/switchbot.markdown:20]: This is not correct.
You only need Bluetooth on the device running Home Assistant

Comment (frenck) [reply source/_integrations/switchbot.markdown:20]: Thanks, @Danielhiversen 👍 

In that case, we'll close up this PR.

Nevertheless, thanks for willing to contribute @tels7ar

**Pull request closed without merging**